presto-docs/src/main/sphinx/release/release-0.218.rst

```diff
@@ -0,0 +1,43 @@
+=============
+Release 0.218
+=============
+
+General Changes
+---------------
+
+* Add support for using binary (smile) encoding for coordinator-to-worker communication.
+This feature is experimental, and it can be enabled with the ``experimental.internal-communication.binary-transport-enabled`` config option.
+Enabling this feature should help with coordinator scalability, and should reduce network, cpu, and memory usage on the coordinator.
+* Add ST_Area for SphericalGeography
+* Add a new system table ``system.metadata.analyze_properties`` to display available
+properties that can be used in ``ANALYZE`` statement.
+* Optimize performance for queries with COALESCE join keys on top of FULL OUTER JOIN
+* Fix failures in regular expression functions for certain inputs when the pattern contained word boundaries.
+* Consider estimated query peak memory when making cost based decisions.
+* Add an optional additional argument to split_to_map to resolve key conflicts
```

Comment:
- `.` at the end.
- Need to link to `split_to_map` doc.
- Add double backticks around split_to_map.